src/bandersnatch/verify.py

```diff
@@ -109,7 +109,15 @@ async def verify(
 
     if args.json_update:
         if not args.dry_run:
-            await get_latest_json(master, json_full_path, config, executor, args.delete)
+            try:
+                await get_latest_json(
+                    master, json_full_path, config, executor, args.delete
+                )
+            except Exception:
```

Comment:
I always recommend (and it's a good practice) catching only the exceptions you expect explicitly. Here I feel it's better to error then keep on going if someone wants new JSON to be downloaded before trying to verify a package.

Do we actually want to continue here? If so, I feel we should respect the config `stop-on-error`

Maybe This?
```suggestion
            except aiohttp.ClientError:
```

Reply:
I generally agree. But I was basing this on the general behaviour of the mirror mode which catches all Exceptions to allow the command to continue. Also, it's not just client error exceptions but also timeouts that I was experiencing, and these would cause the process to stop completely, so just took the approach of log and continue.